packages/dns-forward/dns-forward.0.7.2/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "1.2"
+maintainer:   "dave@recoil.org"
+authors:      ["David Scott"]
+license:      "ISC"
+homepage:     "https://github.com/djs55/ocaml-dns-forward"
+bug-reports:  "https://github.com/djs55/ocaml-dns-forward/issues"
+dev-repo:     "https://github.com/djs55/ocaml-dns-forward.git"
+doc:          "https://github.com/djs55/ocaml-dns-forward"
```

Comment:
I suspect you didn't use `topkg bistro` to do the release otherwise you would have seen an error here :-)

Should be: https://github.com/djs55/ocaml-dns-forward/ (with a trailing slash)